authui/src/authflowv2/alert-message.ts

```diff
@@ -11,6 +11,10 @@ export function handleAxiosError(e: unknown) {
   console.error(err);
 }
 
+/**
+ * This code duplicates with authui/src/setErrorMessage-message.ts. However, alert-message.ts cannot import other modules because it is commonjs script instead
+ * Please help keep code in `setErrorMessage.ts` and `alert-message.ts` sync if you are to make any changes
+ */
```

Comment:
`alert-message.ts` can be imported. For example, it was imported in authui/src/authflowv2/turboForm.ts:6

you can simply export this function without duplicating it.